polymetis/polymetis/torch_isolation/pinocchio_isolation/src/pinocchio_wrapper.cpp

```diff
@@ -166,6 +165,13 @@ int64_t get_link_idx_from_name(State *state, const char *link_name) {
   return result;
 }
 
+char *get_link_name_from_idx(State *state, int64_t link_idx) {
+  if (link_idx >= state->model->nframes) {
```

Comment:
Unit test this, esp. since Pinocchio's nframes is has a slightly different convention iirc.

Reply:
Oops, this functionality is actually only needed for instantiating a trac-IK solver, but I'll add a test for it and keep it in the PR in case it comes in handy someday.